models/google_play__country_report.sql

```diff
@@ -0,0 +1,150 @@
+with installs as (
+
+    select *
+    from {{ var('stats_installs_country') }}
+
+), ratings as (
+
+    select *
+    from {{ var('stats_ratings_country') }}
+
+
+), store_performance as (
+
+    select *
+    from {{ var('stats_store_performance_country') }}
+
+), install_metrics as (
+
+    select
+        *,
+        -- take the difference later or nah?
+        sum(daily_device_installs) over (partition by country, package_name rows between unbounded preceding and current row) as total_device_installs,
+        sum(daily_device_uninstalls) over (partition by country, package_name rows between unbounded preceding and current row) as total_device_uninstalls
+
+    from installs 
```

Comment:
New style standards should keep the `from` clause flush with the last field queried. Can you apply the same updates to the other CTEs and models.
```suggestion
    from installs 
```

Reply:
should i apply this to the source package models as well?

Reply:
That would be great!